src/app/panels/image-panel/image-panel.component.html

```diff
@@ -1,8 +1,9 @@
-<evt-panel
-    [hideHeader]="true"
+<evt-panel [showSecondaryContent]="msDescOpen"
+    [hideHeader]="false"
     [hideFooter]="true">
     <div header-left>
         <!-- TODO: Add dropdowns for navigation -->
```

Comment:
Is the comment still necessary?

Reply:
Yes. We need to remember to add dropdown for page navigation